.github/workflows/publish-android.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         uses: softprops/action-gh-release@v2
         with:
             files: android.zip
+            fail_on_unmatched_files: true
```

Comment:
This is actually important. Crazy that it's not the default. https://github.com/softprops/action-gh-release/issues/348